modules/market/src/main/java/com/opengamma/strata/market/view/ZeroRatePeriodicDiscountFactors.java

```diff
@@ -202,7 +203,12 @@ public ZeroRateSensitivity zeroRatePointSensitivityWithSpread(
   @Override
   public CurveUnitParameterSensitivities unitParameterSensitivity(LocalDate date) {
     double relativeYearFraction = relativeYearFraction(date);
-    return CurveUnitParameterSensitivities.of(curve.yValueParameterSensitivity(relativeYearFraction));
+    double rp = curve.yValue(relativeYearFraction);
+    //    double rc = _compoundingPeriodsPerYear * Math.log(1 + rp / _compoundingPeriodsPerYear);
```

Comment:
Remove this line or elaborate this to an implementation note on the relation between the two rates.